tests/test_install_aktualizr_and_update.sh

```diff
@@ -33,4 +36,4 @@ if [ $? -ne 0 ];then
     echo "ERROR: $filename does not exist or sha256sum does not match."
```

Comment:
I also have a bit of concern at line33-line35. what if somehow `aktualizr_info` run into some error and return a blank filename, and `ls $TEMP_DIR/images/$filename` still return to 0, but actually it shouldn't?

Reply:
yes actually this test is weird, it should just be `if [ ! -f ... ]; then`